libs/angular/table/src/lib/table/ngx-table.component.html

```diff
@@ -251,9 +294,13 @@
 				'ngx-table-row',
 				rowClass || '',
 				row[highlightKey] ? 'ngx-table-row-highlight' : '',
-				first ? 'ngx-table-row-first' : '',
+				first
+					? 'ngx-table-row-first'
+					: !contextRowClasses[dataIndex] && dataIndex === 0
+						? 'ngx-table-row-first'
+						: '',
```

Comment:
These inline if-statements (also in other parts of the code) are getting out of hand. Consider opening an extra issue to split them to utils / consts / ... to preserve legibility